alfasim_sdk/_alfasim_sdk_utils.py

```diff
@@ -72,3 +72,26 @@ def get_all_attributes(class_: type):
         attributes[key] = attr.ib(default=value)
 
     return attributes
+
+
+def get_current_version():
+    """
+    Checks current version of alfasim-sdk. Extracted to be easier to mock in tests.
+    :return:
+    """
+    import alfasim_sdk
+
+    return alfasim_sdk.__version__
+
+
+def get_extras_default_required_version():
+    """
+    :rtype str:
+    :return:
+        Returns a string with default alfasim-sdk version requirement for plugins. Default is
+        greater or equal current version and lesser than next major release.
+    """
+    current_version = get_current_version()
```

Comment:
For strict correctness we should require >= current minor, so bug fixes are not part of the required version:

```python
    parts = current_version.split(".")
    current_major = parts[0]
    current_minor = ".".join(parts[:2])
    next_major = int(current_major) + 1
    return f">={current_minor}, <{next_major}"
```